deployment/scripts/codeship_heroku

```diff
@@ -10,10 +12,15 @@ SCRIPT_NAME=${1:?'You need to provide the action as the first parameter'}
 
 SCRIPT_PATH=$DIR/codeship_heroku_$SCRIPT_NAME
 
-if [ -x $SCRIPT_PATH ]
-then
-  $SCRIPT_PATH "${@:2}"
+if [ -x "${SCRIPT_PATH}" ]; then
+  if [ "${SCRIPT_NAME}" == "deploy" ]; then
```

Comment:
Deprecating the codeship_heroku script for anything other than "deploy". "deploy" calls the new deployment script, matching old params against new ones.